ticker.go

```diff
@@ -18,15 +19,26 @@ type Ticker struct {
 	Symbol      string    `json:"symbol"`
 }
 
+// Tickers rapresents a set of a valid Tickers struct
+type Tickers []Ticker
+
 func (t *Ticker) UnmarshalJSON(data []byte) error {
 	var err error
 	type Alias Ticker
 	aux := &struct {
 		Timestamp string `json:"timestamp"`
+		High      string `json:"high"`
```

Comment:
This is for when high/low is missing?

Reply:
Yes

Reply:
This can be handled omitempty in ticker struct